test/data/outfitters.get.id.json

```diff
@@ -0,0 +1,49 @@
+{
+    "response": {
+        "success" : true,
+        "api": "FS ePermit API",
+        "type": "controller",
+        "verb": "get",
+        "src": "json",
+        "route": "permits/special-uses/commercial/outfitters/id"
+    },
+    "accinstCn": "1095010376",
+    "region": 3,
+    "forest": 50553,
+    "district": 50553,
+    "authorizingOfficerName": "WILLIAM L.NOXON",
+    "authorizingOfficerTitle": null,
+    "applicant-info": {
+      "firstName": null,
+      "lastName": null,
+      "dayPhone": {
+        "areaCode": 541,
+        "number": 8156141,
+        "extension": 0,
+        "type": "BUSINESS"
+      },
+      "eveningPhone": {
+        "areaCode": 0,
+        "number": 0,
+        "extension": 0,
+        "type": null
+      },
+      "emailAddress": "test@email.org",
+      "mailingAddress": "ON ANW 0953",
+      "mailingCity": "ALBANY",
+      "mailingState": "OR",
+      "mailingZIP": 97321,
+      "organizationName": "COOLEY RIVER EXPEDITIONS",
+      "website": null,
+      "orgType": "Limited Liability Company"
+    },
+    "temp-outfitter-fields": {
+      "activityDescription": "PROVIDING WHITEWATER OUTFITTING AND GUIDING ACTIVITIES ON NATIONAL FOREST LANDS",
+      "locationDescription": "string",
+      "startDateTime": "2013-01-12",
+      "endDateTime": "2013-01-19",
+      "insuranceCertificate": "insuranceCertificate.pdf",
+      "goodStandingEvidence": "goodStandingEvidence.pdf",
```

Comment:
insuranceCertificate  should be of file type, goodStandingEvidence and operatingPlan fields will be added by the API, not from this POST request; these needs to be reviews with the client.